src/core/messages.ts

```diff
@@ -396,8 +411,8 @@ export class DefaultMessages
   /**
    * @inheritdoc Messages
    */
-  get channel(): Ably.RealtimeChannel {
```

Comment:
I think you could still call this `channel` - its not usually common to add `promise` to the method name

Reply:
Ditto for the others